assets/js/blocks/reviews/edit-utils.js

```diff
@@ -51,21 +51,24 @@ export const getSharedReviewContentControls = ( attributes, setAttributes ) => {
 					className="wc-block-reviews__notice"
 					isDismissible={ false }
 				>
-					<RawHTML>
-						{ sprintf(
-							escapeHTML(
-								/* translators: A notice that links to WooCommerce settings. */
-								__(
-									'Product rating is disabled in your %sstore settings%s.',
-									'woo-gutenberg-products-block'
-								)
+					{ __experimentalCreateInterpolateElement(
+						__(
+							'Product rating is disabled in your <a>store settings</a>.',
+							'woo-gutenberg-products-block'
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									href={ getAdminLink(
+										'admin.php?page=wc-settings&tab=products'
+									) }
+									target="_blank"
+									rel="noopener noreferrer"
```

Comment:
I misread the linting rule and thought both were required. Referer could be useful (?) so Ill use `noopener`.

Reply:
hmm actually the rule does enforce both. `eslint(react/jsx-no-target-blank)` You cannot just use one or the other...

Reply:
🤔 Interesting, from [this](https://mathiasbynens.github.io/rel-noopener/) it appears this targets both because of older browser support. I can't think of any reason offhand why doing both creates a problem, so let's just keep it as is for now.

Reply:
Looks like there's an issue that got started around this rule enforcing both :)

Reply:
For reference: a [similar discussion](https://github.com/woocommerce/storefront/pull/1243) happened in Storefront repo. In the end we ended up using `rel="noreferrer"`. We don't have this ESLint rule so it was possible to use only one in that case.

Reply:
Thanks for the links. I think we're okay using both to keep eslint happy until that issue is fixed. There doesn't appear to be consequences in doing so.